ca/certificate-authority_test.go

```diff
@@ -187,6 +189,56 @@ func setup(t *testing.T) *testCtx {
 	return &testCtx{ssa, caConfig, reg, pa, fc, cleanUp}
 }
 
+func (ctx *testCtx) importCSR(csr []byte) core.CertificateRequest {
+	return core.CertificateRequest{
+		RegistrationID: ctx.reg.ID,
+		Created:        ctx.fc.Now(),
+		Expires:        ctx.fc.Now().AddDate(1, 0, 0),
+		Status:         core.StatusValid,
+		CSR:            csr,
+	}
+}
+
+// Use this function to wait for the IssueCertificate->sign->SA handoff
+//
+//   Note: No need to use this for negative tests ("Test that the CA
+//   rejects...").  If IssueCertificate() returns without an error, then
+//   the CA has agreed to issue the certificate, which is already a
+//   failure for these cases.
+func (ctx *testCtx) attemptToIssue(t *testing.T, ca *CertificateAuthorityImpl, csr []byte) (cert core.Certificate, found bool) {
```

Comment:
Since we're still actually sync in this PR, this method can probably be deleted, or at least the "wait around" part.